### DISCUSSION TITLE
Absorption refrigeration Components

### DISCUSSION THREAD

**Discussion by tomam28** [oemof/tespy#367] [Ideas]
Hello Tespy-community,

would it be nice to implement Absorption refrigeration cycle? Or is it by now possible? 

Best regards :)

**Comment by fwitte** [comment]
Dear @tomam28,

that would be a very nice addition. It is currently not possible, but there are some ideas for extension/improvement of the fluid property handling (that is currently the main reason, why this does not work). If you are interested to help with that, we can discuss the topic of fluid properties in today's user meeting at 17:00 CEST in https://meet.jit.si/tespy_user_meeting.

Best

Francesco

**Reply by tomam28** [reply]
Dear Francesco,

thanks for answering! I searched for a fluid property handling tool for absorption fluid pairs (for the most common NH3/H20 and H20/LiBr), something like coolprop but couldnt find anything. But I will search in the future for more :)

Best regards
Tom

**Reply by fwitte** [reply]
Hi Tom,

thanks for your reply. At least LiBr is integrated in CoolProp, but it cannot consider heat of solution/dissolution (as far as I know).

Also, with the current implementation of the fluid properties in TESPy it is not possible to have changes in the solution mass fractions. There would be some implementation work on the road, to make that work. If you are interested in contributing, feel free to contact me. I have some things in mind for that but unfortunately not the time to do it at this moment.

best regards 

Francesco

**Reply by fatsen** [reply]
Dear Francesco, I'm new for tespy. Is it now possible to have changes in the solution mass fractions? Thank you.

**Reply by fwitte** [reply]
It is in principle yes, but the routines to calculate mixtures of liquid solution and steam are not quite finished yet. I work on this feature and plan to publish it within the first half of this year.

**Comment by defencedog** [comment]
LiBr absorption chillers are the most popular

**Comment by kiselaTruba** [comment]
Hello @fwitte, could you say what is the current status of absorption process implementation in tespy? Are there any new developments?

Thank you and best regards

**Reply by fwitte** [reply]
Hi @kiselaTruba, no updates but I invite you to support with the implementation. There is an ancient proof of concept, which did work. Just needs to be updated an refined. Write me an email if you are interested

**Comment by fwitte** [comment]
I reopenend the topic here #1016 and here https://github.com/fwitte/tespy-sorption-models. If anybody is interested, you can push it, give feedback, etc. :)
